doc/quickstart.md

```diff
@@ -0,0 +1,139 @@
+# The Quick Start Guide to Electrum Rust Server
+
+### Prepare bitcoind
+
+In your bitcoin.conf (usually sitting in /home/username/.bitcoin) ensure you have the following:
+```
+rpcallowip=127.0.0.1
+rpcallowip=10.0.0.0/8
+rpcallowip=172.0.0.0/8
+rpcallowip=192.0.0.0/8
+rpcuser=bitcoin
+rpcpassword=bitcoin
+whitelist=127.0.0.1
+whitelist=download@127.0.0.1
+```
+Note: Be sure to restart your bitcoind instance for changes to take effect.
+Your rpcuser and rpcpassword can be set to whatever you want, I've used bitcoin/bitcoin here.
+
+
+### Prepare system to build electrs
+
+```
+sudo apt update
+sudo apt install clang cmake build-essential
+```
+
+### Install latest version of Rust
+
+`curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh`
+
+### Download electrs
+
+In your home directory, download the electrs github repository and change directory into it.
+
+`cd ~`
```

Comment:
nit: consider using triple-quotes here (and below) as well.

Reply:
agree